lang/cs/Org.Apache.REEF.Client/YARN/YarnREEFDotNetClient.cs

```diff
@@ -146,6 +146,17 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
             return application.FinalStatus;
         }
 
+        /// <summary>
+        /// Returns all the application reports running in the cluster
+        /// </summary>
+        /// <returns></returns>
+        /// TODO: [REEF-1825]: Implement GetApplicationReports to return the status of the applications.
+        public async Task<IList<IApplicationReport>> GetApplicationReports()
```

Comment:
Internally it uses YarnRMClient. It should be similar. As we already have a Jira for it, let's do it separately.